tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -158,6 +159,11 @@ public IScope ActiveScope
             }
         }
 
+        /// <summary>
+        /// Gets the active span context dictionary by consulting DistributedTracer.Instance
+        /// </summary>
+        internal IReadOnlyDictionary<string, string> DistributedSpanContext => DistributedTracer.Instance.GetSpanContextRaw() ?? InternalActiveScope?.Span?.Context;
```

Comment:
I don't think the `??` fallback is needed here, but we can leave it just in case